addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2302,9 +2302,23 @@ var SnippetsMenu = Widget.extend({
                     self.options.wysiwyg.odooEditor.automaticStepSkipStack();
                     $toInsert.removeClass('oe_snippet_body');
                     self.draggableComponent.$scrollTarget.off('scroll.scrolling_element');
-
+                    let $el;
```

Comment:
mtnt que un peu plus lisible sur le diff gh, le $el n'est plus necessaire, le dropperOnSharedZone pdevrai etre reltaed to 'NotNearest' et le code semple un peu verbeux vu que 2 fois presue identique.